helm/cluster-api-core/templates/webhook-deployment.yaml

```diff
@@ -1,65 +1,63 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  name: {{ include "resource.webhook.name" . }}
+  namespace: {{ include "resource.default.namespace" . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
-  name: {{ include "resource.default.name" . }}-webhook
-  namespace: {{ include "resource.default.namespace" . }}
 spec:
   replicas: 1
   selector:
     matchLabels:
       {{- include "labels.selector" . | nindent 6 }}
   template:
     metadata:
+      annotations:
+        releaseRevision: {{ .Release.Revision | quote }}
       labels:
         {{- include "labels.common" . | nindent 8 }}
     spec:
       containers:
       - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.1
```

Comment:
AFAICT this is needed so prometheus can scrape the controller. If so, I dropped it because I wasn't sure if there were any useful metrics to scrape when running in webhook mode.

Reply:
Can we keep it for now? We are still trying to figure out metrics collection properly :D

Reply:
It's back now. I wasn't sure about the service ports, but it should be workable.